spec/jsSpecs/ares-package.spec.js

```diff
@@ -522,8 +602,12 @@ describe(aresCmd + ' negative TC for services packaging', function() {
 
     it('Check to exist id fields in pkg meta file', function(done) {
         exec(cmd + ` ${sampleServicePaths[1]} ${sampleServicePaths[2]} -pf ${pkginfoPath} -o ${outputPath}`, function (error, stdout, stderr) {
-            stderr.trim().replace(/\s+['\n']/g, '\n');
-            expect(stderr).toContain("ares-package ERR! CLI: Please input required field <id>", error);
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+                stderr.trim().replace(/\s+['\n']/g, '\n');
+                console.log("stderr:"+stderr)
```

Comment:
If you don't need this console.log, please remove it.

If you want to keep it, please add "space" such as
console.log("stderr: " + stderr);

Reply:
Sorry this is my mistake. I've removed this